import

```diff
@@ -492,6 +506,57 @@ var updatePlace = function(obj) {
   }, stats.rows * options.wait);
 };
 
+var importRecord = function(obj, type) {
+  var config = supportedTypes[options.recordType],
+      record = {
+        imported_date: new Date().toISOString(),
+      };
+  Object.keys(obj).forEach(function(key) {
+    if (key === 'uuid') {
+      record._id = obj[key];
+    } else if (key === 'reported_date') {
+      record.reported_date = Number(obj[key]);
+    } else {
+      record[key] = obj[key];
+    }
+  });
+  record = flat.unflatten(record); // support dot notation in keys
+  record.type = config.doc_type;
+  record.content_type = type || 'xml';
+  setTimeout(function() {
+    reqOpts.method = 'POST';
```

Comment:
Yes this needs to be updated with bulk docs support (PouchDB lib probably) at some point.  The setTimeout is pretty much a hack to spread out requests over time.